code/modules/projectiles/projectile/beams.dm

```diff
@@ -97,7 +97,13 @@
 	hitsound = 'sound/weapons/tap.ogg'
 	eyeblur = 0
 	impact_effect_type = /obj/effect/temp_visual/impact_effect/blue_laser
-	light_color = LIGHT_COLOR_CYAN
+	light_color = LIGHT_COLOR_LIGHT_CYAN
```

Comment:
Worth adding to the PR description that the disabler beam color is now slightly different.

Reply:
Why are we even changing it for normal disablers?

Reply:
Yet another mistake from when I was testing something, thanks for catching it out. Changing it back in next commit.